code/modules/food_and_drinks/recipes/soup_mixtures.dm

```diff
@@ -1757,3 +1757,402 @@
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/new_osaka_sunrise
+	name = "\improper New Osaka Sunrise soup"
+	icon = 'icons/obj/food/martian.dmi'
+	icon_state = "new_osaka_sunrise"
+	drink_type = MEAT | GRAIN | DAIRY | VEGETABLES
+
+/datum/chemical_reaction/food/soup/new_osaka_sunrise
+	required_reagents = list(
+		/datum/reagent/consumable/nutriment/soup/miso = 15,
+	)
+	required_ingredients = list(
+		/obj/item/food/grown/herbs = 1,
+		/obj/item/food/grown/eggplant = 1,
+		/obj/item/food/onion_slice = 1,
+		/obj/item/food/tofu = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/new_osaka_sunrise = 30,
+		/datum/reagent/consumable/nutriment/vitamin = 8,
+		/datum/reagent/consumable/nutriment/protein = 2,
+	)
+
+// Satsuma Black
+/datum/reagent/consumable/nutriment/soup/satsuma_black
+	name = "Satsuma Black Soup"
+	description = "A rich, heavy seafood and noodle soup from Mars, employing squid ink to give a strong taste of the sea."
+	data = list("seafood" = 1, "tofu" = 1, "noodles" = 1)
+	color = "#F0DD5A"
+
+/datum/glass_style/has_foodtype/soup/satsuma_black
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/satsuma_black
+	name = "\improper Satsuma Black soup"
+	icon = 'icons/obj/food/martian.dmi'
+	icon_state = "satsuma_black"
+	drink_type = SEAFOOD | GRAIN | VEGETABLES
+
+/datum/chemical_reaction/food/soup/satsuma_black
+	required_reagents = list(
+		/datum/reagent/consumable/nutriment/soup/dashi = 20,
+	)
+	required_ingredients = list(
+		/obj/item/food/spaghetti/boilednoodles = 1,
+		/obj/item/food/seaweedsheet = 1,
+		/obj/item/food/tofu = 1,
+		/obj/item/food/canned/squid_ink = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/satsuma_black = 30,
+		/datum/reagent/consumable/nutriment/vitamin = 4,
+		/datum/reagent/consumable/nutriment/protein = 6,
+	)
+
+// Dragon Style
+/datum/reagent/consumable/nutriment/soup/dragon_ramen
+	name = "Dragon Style Ramen"
+	description = "For the ramen fan who hates their tastebuds and digestive tract. Traditionally made with seven different chilis, although after two or so the point sorta gets lost."
+	data = list("meat" = 1, "liquid hot magma" = 1, "noodles" = 1)
+	color = "#F0DD5A"
+
+/datum/glass_style/has_foodtype/soup/dragon_ramen
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/dragon_ramen
+	name = "\improper Dragon Style ramen"
+	icon = 'icons/obj/food/martian.dmi'
+	icon_state = "dragon_ramen"
+	drink_type = SEAFOOD | GRAIN | VEGETABLES
+
+/datum/chemical_reaction/food/soup/dragon_ramen
+	required_reagents = list(
+		/datum/reagent/consumable/nutriment/soup/dashi = 20,
+		/datum/reagent/consumable/nutriment/soup/teriyaki = 10,
+		/datum/reagent/consumable/red_bay = 5,
+	)
+	required_ingredients = list(
+		/obj/item/food/spaghetti/boilednoodles = 1,
+		/obj/item/food/grown/ghost_chili = 2,
+		/obj/item/food/kamaboko_slice = 1,
+		/obj/item/food/boiledegg = 1,
+	)
```

Comment:
Fair warning, two ghost chilis will completely dominate the recipe result of this with botany produce. 
Bearing in mind 1 ghost chili contains `30 units of capsaicin, 55 units of condensed capsaicin` 
May want to bump the `ingredient_reagent_multiplier` way down to compensate.